### PR TITLE
Elgato.CameraHub version 1.3.0.596

### PR DESCRIPTION
**Pull request by sanyer**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/67534)

### CODE REVIEW

File: manifests/e/Elgato/CameraHub/1.3.0.596/Elgato.CameraHub.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Elgato.CameraHub
+PackageVersion: 1.3.0.596
+PackageLocale: en-US
+Publisher: Elgato Systems
```
Comment (ItzLevvie) [inline manifests/e/Elgato/CameraHub/1.3.0.596/Elgato.CameraHub.locale.en-US.yaml:7]:
```suggestion
Publisher: Elgato
```
This must match Control Panel.

File: manifests/e/Elgato/CameraHub/1.3.0.596/Elgato.CameraHub.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Elgato.CameraHub
+PackageVersion: 1.3.0.596
+PackageLocale: en-US
+Publisher: Elgato Systems
+PublisherUrl: https://www.elgato.com/en
+PrivacyUrl: https://www.elgato.com/en/privacy-policy
+Author: Elgato Systems
+PackageName: Camera Hub
```
Comment (ItzLevvie) [inline manifests/e/Elgato/CameraHub/1.3.0.596/Elgato.CameraHub.locale.en-US.yaml:11]:
```suggestion
PackageName:  Elgato Camera Hub
```
This must match Control Panel.

File: manifests/e/Elgato/CameraHub/1.3.0.596/Elgato.CameraHub.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerType: wix
+  InstallerUrl: https://edge.elgato.com/egc/windows/echw/1.3/CameraHub_1.3.0.596_x64.msi
+  InstallerSha256: D16E333102BEACFD25B3D58E3F5177160076066A54DC64FB4C594FD046BD040F
+  ProductCode: '{8C4B73DD-C240-439F-9DAD-B1C621E936FC}'
```
Comment (ItzLevvie) [inline manifests/e/Elgato/CameraHub/1.3.0.596/Elgato.CameraHub.installer.yaml:20]:
```suggestion
  ProductCode: '{8C4B73DD-C240-439F-9DAD-B1C621E936FC}'
  AppsAndFeaturesEntries:
  - DisplayVersion: 1.3.0.595
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by ItzLevvie** [comment]
Azure Pipelines is running into this error - but works fine locally on WinINet:
```
2022-07-26 02:02:55.037 [CORE] DeliveryOptimization downloading from url: https://edge.elgato.com/egc/windows/echw/1.3/CameraHub_1.3.0.596_x64.msi
2022-07-26 02:02:57.441 [CORE] DeliveryOptimization error: 0x80190193, extended error: 0x00000000
2022-07-26 02:02:57.443 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\DODownloader.cpp(289)\WindowsPackageManager.dll!00007FF86C65DA8B: (caller: 00007FF86C65E1CB) Exception(1) tid(10d8) 80190193 Forbidden (403).

2022-07-26 02:02:57.881 [CORE] Started applying motw to C:\Users\VALIDA~1\AppData\Local\Temp\WinGet\Elgato.CameraHub.1.3.0.596\d16e333102beacfd25b3d58e3f5177160076066a54dc64fb4c594fd046bd040f with zone: 3
2022-07-26 02:02:57.897 [CORE] Finished applying motw
2022-07-26 02:02:57.897 [CORE] WinINet downloading from url: https://edge.elgato.com/egc/windows/echw/1.3/CameraHub_1.3.0.596_x64.msi
2022-07-26 02:02:58.186 [CORE] Download request failed. Returned status: 403
2022-07-26 02:02:58.186 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Downloader.cpp(62)\WindowsPackageManager.dll!00007FF86C61C156: (caller: 00007FF86C61C5B4) Exception(2) tid(10d8) 80190193 Forbidden (403).
    Msg:[Download request status is not success.] 

2022-07-26 02:02:58.187 [CLI ] Failed to download, waiting a bit and retry. Url: https://edge.elgato.com/egc/windows/echw/1.3/CameraHub_1.3.0.596_x64.msi
2022-07-26 02:02:58.688 [CORE] Downloading to path: C:\Users\VALIDA~1\AppData\Local\Temp\WinGet\Elgato.CameraHub.1.3.0.596\d16e333102beacfd25b3d58e3f5177160076066a54dc64fb4c594fd046bd040f
2022-07-26 02:02:58.688 [CORE] DeliveryOptimization downloading from url: https://edge.elgato.com/egc/windows/echw/1.3/CameraHub_1.3.0.596_x64.msi
2022-07-26 02:02:58.816 [CORE] DeliveryOptimization error: 0x80190193, extended error: 0x00000000
2022-07-26 02:02:58.816 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\DODownloader.cpp(289)\WindowsPackageManager.dll!00007FF86C65DA8B: (caller: 00007FF86C65E1CB) Exception(3) tid(10d8) 80190193 Forbidden (403).

2022-07-26 02:02:58.840 [CORE] Started applying motw to C:\Users\VALIDA~1\AppData\Local\Temp\WinGet\Elgato.CameraHub.1.3.0.596\d16e333102beacfd25b3d58e3f5177160076066a54dc64fb4c594fd046bd040f with zone: 3
2022-07-26 02:02:58.841 [CORE] Finished applying motw
2022-07-26 02:02:58.841 [CORE] WinINet downloading from url: https://edge.elgato.com/egc/windows/echw/1.3/CameraHub_1.3.0.596_x64.msi
2022-07-26 02:02:58.908 [CORE] Download request failed. Returned status: 403
2022-07-26 02:02:58.908 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Downloader.cpp(62)\WindowsPackageManager.dll!00007FF86C61C156: (caller: 00007FF86C61C5B4) Exception(4) tid(10d8) 80190193 Forbidden (403).
    Msg:[Download request status is not success.] 

2022-07-26 02:02:58.914 [CLI ] Caught wil::ResultException: D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Downloader.cpp(62)\WindowsPackageManager.dll!00007FF86C61C156: (caller: 00007FF86C61C5B4) Exception(4) tid(10d8) 80190193 Forbidden (403).
    Msg:[Download request status is not success.] 
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/67534#pullrequestreview-1053347703

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**